examples/step_operator_remote_training/run.py

```diff
@@ -4,14 +4,14 @@
 #  you may not use this file except in compliance with the License.
 #  You may obtain a copy of the License at:
 #
-#       https://www.apache.org/licenses/LICENSE-2.0
+#       http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Any reason for this?

Reply:
No it must have been some copy paste thing